storage-transcode-videos/functions/.eslintrc.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
```

Comment:
can/should this be in the root and used by all extensions?

Reply:
Probably yes, I think this should be addressed after merge so as not to scope creep here